src/app/layouts/SimplePageLayout.tsx

```diff
@@ -6,34 +6,50 @@ import DocBg from 'app/a11y/DocBg';
 import Logo from 'app/atoms/Logo';
 import { useAppEnv } from 'app/env';
 import ContentContainer from 'app/layouts/ContentContainer';
+import ChristmasBgPopupImg from 'app/misc/christmas-popup-bg.png';
 
 interface SimplePageLayoutProps extends PropsWithChildren {
   title: ReactNode;
+  isConfirmationPopup?: boolean;
 }
 
-const SimplePageLayout: FC<SimplePageLayoutProps> = ({ title, children }) => {
-  const appEnv = useAppEnv();
+const SimplePageLayout: FC<SimplePageLayoutProps> = ({ isConfirmationPopup = false, title, children }) => {
+  const { popup } = useAppEnv();
 
   return (
     <>
-      {!appEnv.fullPage && <DocBg bgClassName="bg-primary-white" />}
+      {popup && <DocBg bgClassName="bg-primary-white" />}
 
-      <ContentContainer className={classNames('min-h-screen', 'flex flex-col', !appEnv.fullPage && 'bg-primary-white')}>
+      <ContentContainer
+        className={classNames(
+          'min-h-screen',
+          'flex flex-col',
+          popup && !isConfirmationPopup && 'bg-gray-100 bg-no-repeat'
+        )}
+        style={{ backgroundImage: isConfirmationPopup ? undefined : `url(${ChristmasBgPopupImg})` }}
+      >
         <div className={classNames('mt-12 mb-10', 'flex flex-col items-center justify-center')}>
-          <div className="flex items-center">
-            <Logo hasTitle />
+          <div className="flex items-center bg-gray-100">
+            <Logo hasTitle style={{ height: 64, marginTop: 0, marginBottom: 0 }} />
```

Comment:
this will not work, cause styles from Logo component will overwrite this class styles. Also those changes will be reverted after holidays btw, so don't care about them too much.